### ISSUE TITLE
Availability of the XML for 'peilmetingen'

### ISSUE THREAD

**Issue by stijnvanhoey** [DOV-Vlaanderen/pydov#14]
@johanvdw is the service resulting in the XML-file for the peilmetingen, cfr. https://github.com/DOV-Vlaanderen/pydov/blob/master/tests/data/1-0709.xml yet available? Could you provide some links to documentation (or just the URL and query params to use)? Hence, we can schedule the further implementation and documentation for other users.

**Comment by Roel** [comment]
Yes, the application and corresponding XML export has been available for some weeks now. Recently I also updated the 'Grondwatermeetnetten' map layer, which now includes the permalinks to the (XML of) groundwaterlocations and groundwaterfilters.

You can find more information about the layer 'Grondwatermeetnetten' in its [metadata record](https://www.dov.vlaanderen.be/geonetwork/?uuid=6c39d716-aecc-4fbc-bac8-4f05a49a78d5) and about its attributes in the corresponding [feature catalogue](https://www.dov.vlaanderen.be/geonetwork/?uuid=b142965f-b2aa-429e-86ff-a7cb0e065d48).

I also found some time to write a little 'starting guide' on how to access our data, listing service endpoints for example. Pull request is coming soon.
